assets/app/view/game/unsold_companies.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'view/game/company'
+
+module View
+  module Game
+    class UnsoldCompanies < Snabberb::Component
```

Comment:
same question

Reply:
The opening auction has players given a pile of companies that they auction off. This allows people to "own" companies that are waiting to be sold. Probably not applicable many other games, but not able to be represented on screen with any current view options.

Reply:
does it need to be represented? can you just have an italic or something?